Sources/NIOSSH/SSHServerConfiguration.swift

```diff
@@ -23,9 +23,30 @@ public struct SSHServerConfiguration {
     /// The host keys for this server.
     public var hostKeys: [NIOSSHPrivateKey]
 
-    public init(hostKeys: [NIOSSHPrivateKey], userAuthDelegate: NIOSSHServerUserAuthenticationDelegate, globalRequestDelegate: GlobalRequestDelegate? = nil) {
+    /// The ssh banner to display to clients upon authentication
+    public var banner: UserAuthBanner?
+
+    public init(hostKeys: [NIOSSHPrivateKey], userAuthDelegate: NIOSSHServerUserAuthenticationDelegate, globalRequestDelegate: GlobalRequestDelegate? = nil, banner: UserAuthBanner? = nil) {
         self.hostKeys = hostKeys
         self.userAuthDelegate = userAuthDelegate
         self.globalRequestDelegate = globalRequestDelegate ?? DefaultGlobalRequestDelegate()
+        self.banner = banner
+    }
+}
+
+// MARK: - UserAuthBanner
+public extension SSHServerConfiguration {
```

Comment:
Minor nit: NIO style prefers to put the accessors on the things _within_ the extension, not the extension itself.